source/renderer/app/components/wallet/settings/PublicKeyQRCodeDialog.js

```diff
@@ -1,38 +1,34 @@
 // @flow
 import React, { Component } from 'react';
 import { observer } from 'mobx-react';
-import { defineMessages, intlShape } from 'react-intl';
+import { intlShape } from 'react-intl';
 import CopyToClipboard from 'react-copy-to-clipboard';
 import SVGInline from 'react-svg-inline';
 import QRCode from 'qrcode.react';
+import { PopOver } from 'react-polymorph/lib/components/PopOver';
 import DialogCloseButton from '../../widgets/DialogCloseButton';
 import Dialog from '../../widgets/Dialog';
 import iconCopy from '../../../assets/images/clipboard-ic.inline.svg';
-import styles from './WalletPublicKeyQRCodeDialog.scss';
+import styles from './PublicKeyQRCodeDialog.scss';
 import globalMessages from '../../../i18n/global-messages';
 
-const messages = defineMessages({
-  dialogTitle: {
-    id: 'wallet.settings.walletPublicKey',
-    defaultMessage: '!!!Wallet Public Key',
-    description: 'Title for the "Wallet Public Key QR Code" dialog.',
-  },
-  copyPublicKeyLabel: {
-    id: 'wallet.settings.copyPublicKey',
-    defaultMessage: '!!!Copy public key',
-    description: 'Copy public key label.',
-  },
-});
+export type $npm$ReactIntl$MessageDescriptor = {
```

Comment:
Why such a cryptic type name? And why is it exported from this component if it's a generic type?